src/App.tsx

```diff
@@ -1,222 +1,273 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+
+import classNames from 'classnames';
 import { AuthContext } from './components/Auth/AuthContext';
+import {
+  getTodos, sendTodos, deleteTodos, patchTodos,
+} from './api/todos';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Todo } from './types/Todo';
+import { SendedTodo } from './types/SendedTodo';
+
+const positionValues = ['All', 'Active', 'Completed'];
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const [newTodo, setNewTodo] = useState<SendedTodo | null>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [position, setPosition] = useState('all');
+  const [currTodoId, setCurrTodoId] = useState(0);
+  const [hidden, setHidden] = useState(true);
+  const [error, setError] = useState('');
+  const [isForm, setIsForm] = useState(false);
+
+  const loadTodos = async () => {
+    if (user) {
+      try {
+        const todosFromApi = await getTodos(user?.id);
+
+        setTodos(todosFromApi);
+      } catch {
+        setHidden(false);
+        setError('Todos not found');
+        throw new Error('Todos not found');
+      }
+    }
+  };
+
+  async function sendNewTodo(data: SendedTodo) {
```

Comment:
```suggestion
  const postTodo = async (data: SendedTodo) => {
```
it would be better, also let's not mix function declaration and arrow function in one project

Reply:
check all such cases